utils/envvars/parser.go

```diff
@@ -163,22 +161,22 @@ func parseStringDuration(value string) (time.Duration, error) {
 
 // GetSAASHost returns SaaS host env variable value if it's present and valid.
 // Otherwise returns defaultSaaSHost.
-func GetSAASHost(fallbackEnv string) (string, error) {
+func GetSAASHost() (string, error) {
 	name, v := envSaaSHost, os.Getenv(envSaaSHost)
 	if v == "" {
-		name, v = fallbackEnv, os.Getenv(fallbackEnv)
-	}
-	if v == "" {
-		logrus.Debugf("Using default SaaS host %q.", defaultSaaSHost)
+		logrus.Infof("Using default SaaS host %q.", defaultSaaSHost)
 		return defaultSaaSHost, nil
 	}
+	if strings.HasPrefix(v, ":") {
```

Comment:
Why we need that validation?

Reply:
this is for eliminating values like `:443`. Because we are assigning value to `host` if `host` is empty